### ISSUE TITLE
Nodes not displayed in queries if parent(s) satifisy {not: } conditions

### ISSUE THREAD

**Issue by alaq** [Roam-Research/issues#339]
If you're using a query to display every node that have a certain tag, let's call it [[RandomTag]] but you're hiding the [[DONE]] ones, your query would be: `{{[[query]]: {and: [[RandomTag]] {not: [[DONE]] }} }}`
However, if a DONE node has a child node which has the same tag, it will not be shown in the query, as if the child node is inheriting the parent's attributes.

![roam-query-issue](https://user-images.githubusercontent.com/1746118/84465805-f4eb3e80-ac45-11ea-97fb-83ce95af9c50.png)

To see it happening, [here is a link](https://roamresearch.com/#/app/lacqad/page/pi0QUoW7E) to a read-only page

I am not sure if this constitutes a bug but this is confusing for the user who may expect to see the child nodes in the query

**System Information:**
 - Roam version: 0.6.2-f8f8ea57
 - Device: all
 - OS: all
 - Browser: all

**Comment by Awannaphasch2016** [comment]
I don't think this is a bug. Children block inherited Parent's tags as its own.
